### DISCUSSION TITLE
I have no idea what im doing

### DISCUSSION THREAD

**Discussion by TheKoolKid1O1** [community/community#23807] [New to GitHub]
<p>so i wanted to run the program from this <a href="https://github.com/TomWildenhain/MagicPNG" rel="nofollow noopener">https://github.com/TomWildenhain/MagicPNG</a> but there was no .exe file and so i read somewhere that i was supposed to run the sln file but i have no idea how to do that so please help me</p>

**Comment by yamiacat** [comment, marked as answer]
<p><code>.sln</code> files are meant to be opened in Visual Studio, so you might be able to load it in Visual Studio and compile an <code>.exe</code> from there?</p>
<p>In general, when you need help with someone’s project on GitHub, there are a few places we suggest you look:</p>
<ul>
<li>its SUPPORT or README file, which may include contact information or an official website</li>
<li>its Issues tab, where you can report bugs</li>
<li>the owner’s GitHub profile, which may include contact information</li>
</ul>
